openprescribing/frontend/views/mailchimp_utils.py

```diff
@@ -0,0 +1,48 @@
+import hashlib
+import logging
+
+from django.conf import settings
+from mailchimp3 import MailChimp
+from mailchimp3.mailchimpclient import MailChimpError
+
+from common.utils import get_env_setting
+
+logger = logging.getLogger(__name__)
+
+
+def mailchimp_subscribe(
+        request, email, first_name, last_name,
+        organisation, job_title):
+    """Subscribe `email` to newsletter.
+
+    Returns boolean indicating success
+    """
+    del(request.session['newsletter_email'])
+    email_hash = hashlib.md5(email).hexdigest()
+    data = {
+        'email_address': email,
+        'status': 'subscribed',
+        'merge_fields': {
+            'FNAME': first_name,
+            'LNAME': last_name,
+            'MMERGE3': organisation,
+            'MMERGE4': job_title
+        }
+    }
+    client = MailChimp(
+        mc_user=get_env_setting('MAILCHIMP_USER'),
```

Comment:
Do you know why these aren't in `settings`? I'm guessing it's because the MailChimp client used to grab them directly out the environment and then stopped doing that and this was the smallest modification that made it work. That's fine and I don't think it has to be changed immediately, but I think it would be nicer if was done through settings like everything else.